sphinx/source/docs/dev_guide/documentation.rst

```diff
@@ -3,39 +3,104 @@
+documentation. This section also includes details about how to `build`_ and
+`edit`_ the documentation in your local development environment.
 
-Install Requirements
-~~~~~~~~~~~~~~~~~~~~
+.. _`style guidelines`:
 
-In order to build the docs from source, it is recommended that you first
-follow the instructions in :ref:`devguide_setup`.
+Documentation style guidelines
+------------------------------
 
-Some of the Bokeh examples in the documentation rely on sample data that is
-not included in the Bokeh GitHub repository or released packages, due to
-their size.
+Bokeh's documentation uses the `Google developer documentation style guide`_.
 
-Once Bokeh is installed, the sample data can be obtained by executing the
-following command at a Bash or Windows prompt:
+If your contribution includes substantial edits or additions, please
+familiarize yourself with Google's style guide. A simple way to get started
+is using Google's free `technical writing courses`_.
+
+Principles of good style
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow these basic guidelines in all your writing for Bokeh:
+
+.. vale off
+
+* Be aware of **sentence length**. Try to avoid sentences that require more
+  than two commas. Consider breaking up longer sentences. You could also use
+  bulleted or numbered lists instead.
+* Avoid jargon or uncommon words and phrases. Keep in mind that many users of
+  Bokeh don't use English as their primary language.
+* Use **active voice** instead of passive voice whenever possible.
+* Address the reader in the **second person ('you')**. Avoid using the first
+  person ('we') or the third person ('the user').
+* Use **American English** spelling and grammar (refer to `Merriam-Webster`_ for
+  consistent spelling)
+* Use **sentence case for headlines** (capitalize words like in a regular
+  sentence, but do not use any punctuation at the end).
+* Use **serial commas**, also known as Oxford Commas.
+* Write in a way that is inclusive_ and accessible_.
+* Activate the **spell checker** in your development environment.
+
+.. vale on
+
+Refer to the `Google developer documentation style guide`_ for more detailed
+information.
+
+Commonly used terms
+~~~~~~~~~~~~~~~~~~~
+
+These are some commonly used terms and phrases and their spellings used
+throughout the narrative documentation:
+
+.. csv-table::
+   :header: "Term", "Details"
+   :widths: 25, 75
+
+   "Bokeh, BokehJS", "Always capitalize Bokeh and BokehJS"
+   "JavaScript", "Capitalize both 'J' and 'S'"
+   "Jupyter notebook", "Capitalize Jupyter, but not notebook"
+   "Python", "Always capitalize Python (the language)"
+   "[TBD]", "[TBD]"
+
+In general, see the `word list of the Google developer documentation style
+guide`_ for reference.
+
+.. _`build`:
+
+Setting up and building Bokeh's documentation
+---------------------------------------------
+
+Bokeh uses Sphinx_ to generate the HTML files displayed at docs.bokeh.org_. The
+documentation is written in reStructuredText_ (RST).
```

Comment:
Usually the ReST is the abbreviation used, AFAICT

Reply:
I guess Wikipedia states RST, ReST, or reST are all used but ReST is 100% my preference and certainly what I have encountered most ing the scipy/pydata corner of Python

https://en.wikipedia.org/wiki/ReStructuredText

Reply:
Yes, I have seen all three options out in the wild - hard to say which one is "official", so I'm happy to go with ReSt! 😃

Reply:
> with ReSt!

Just clarifying I've seen ReST, but not ReSt

Reply:
sorry, my typo!